core/src/main/scala/scheduler/Kubectl.scala

```diff
@@ -0,0 +1,12 @@
+package nelson
+package scheduler
+
+import cats.effect.IO
+
+object Kubectl {
```

Comment:
Might be work a bit of documentation here. How do these end up becoming `sys.process`?

Reply:
Are you asking how we're actually going to shell out here? If so I am entertaining two options atm, both from the venerable @tpolecat : 

- FS2 approach: https://gist.github.com/tpolecat/f7fac786e084d8e0e2f642b1e8646513
- vanilla IO approach: https://github.com/tpolecat/ocs3/blob/develop/modules/ctl/src/main/scala/low/io.scala

Reply:
I'd probally go for the vanilla I/O approach as its more obvious - we got into trouble doing fancy I/O stuff with streams in funnel, as it was far harder to maintain and understand semantics.